api/api.go

```diff
@@ -109,8 +109,8 @@ func (c *Client) Submissions() (map[string][]SubmissionInfo, error) {
 	return payload, nil
 }
 
-// Submission gets the latest submitted exercise for the given language track id and problem slug.
-func (c *Client) Submission(trackID, slug string) (*Submission, error) {
+// SubmissionURL gets the url of the latest iteration on the given language track id and problem slug.
+func (c *Client) SubmissionURL(trackID, slug string) (*Submission, error) {
```

Comment:
Are we sure this is only returning the URL? We're actually returning a `Submission` struct so it contains all those fields.